Code/Source/UI/gdscInit.cxx

```diff
@@ -287,7 +287,7 @@ int SimVascular_Init( Tcl_Interp *interp )
 #endif
 
 #ifdef USE_PARASOLID
-#ifndef USE_PARASOLID_SHARED
+#if USE_PARASOLID_SHARED == 0

```

Comment:
I think this is what you were asking me about today.  You can use ifdef, this variable will not be defined with the cmake define. #cmakedefine SOME_DEFINE @SOME_DEFINE@ will become /* #undef SOME_DEFINE */ if not defined.

Here is the ZLIB def that is defined in the same file:
https://github.com/SimVascular/SimVascular/blob/b730da692c3c69d716fc7b0153f2327b5f76e548/Code/Source/PostProcessing/cvConvertVisFiles.h#L46